### PR TITLE
show scheduled status in edit modal

### LINKED ISSUE #2006: Show scheduled status in edit modal
**Is your feature request related to a problem? Please describe.**
It's very annoying when you fill in a duration in the Goal edit modal, but have no idea when it will be scheduled.

**Describe the solution you'd like**
Whenever you fill in hours for a Goal, trigger the scheduler (async) and give feedback at the bottom of the edit modal, either:  
- "Auto scheduled"
- "! Impossible"
- "Scheduled someday" (if not scheduled in the next 7 days)

**Describe alternatives you've considered**
A method to manually schedule. This is something that will be necessary - but for a separate issue - to keep things small. Implementing this requirea a lot of design and experimentation as the whole idea of the scheduler is to automatically schedule, given constraints.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #2006

### CODE REVIEW

**Review by tijlleenders — DISMISSED** [review]
Excellent!

**Comment by tijlleenders** [comment]
@Tushar-4781 OK with the code?

File: src/components/GoalsComponents/GoalConfigModal/ConfigGoal.scss [hunk 1]
```diff
@@ -156,3 +156,9 @@
   gap: 10px;
   justify-content: center;
 }
+
+.schedule-status {
+  margin-top: 8px;
+  font-size: 14px;
```
Comment (Tushar-4781) [inline src/components/GoalsComponents/GoalConfigModal/ConfigGoal.scss:162]: use font class from short.scss

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

**Comment by vinaybadgujar102** [comment]
@Tushar-4781 bump

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**